app/src/main/assets/GimelGimel/VectorLayer.js

```diff
@@ -216,6 +211,49 @@ var symbolToBillboardSymbol = function (symbol) {
     }
 };
 
+var createEntityFromSymbol = function (id, location, symbol) {
```

Comment:
this method creates entity from both symbol and location, and not symbol alone. !fix